src/nvim/generators/gen_api_ui_events.lua

```diff
@@ -80,6 +80,77 @@ function call_ui_event_method(output, ev)
   output:write(');\n')
 end
 
+function make_raw_line_and_call(output)
+  -- arg_1 = grid, arg_2 = row, arg_3 = startcol, arg_4 = cells, arg_5 = endcol, arg_6 = clearcol, arg_7 = clearattr
+  
+  output:write([[
+  Integer arg_5, arg_6, arg_7;
```

Comment:
we try to avoid having big chunks of C code in the generator. If possible, put most of the C code in a `.c` file which then includes a generated `.h` file.

Reply:
Sure. I'll do that tomorrow. This was just get things working. Now that I know things work, I'll start making things better and prettier. :)